conf/fred_hutch.config

```diff
@@ -0,0 +1,83 @@
+params {
+    config_profile_name        = 'Fred Hutch'
+    config_profile_description = 'Fred Hutch Cancer Center HPC profile'
+    config_profile_contact     = 'dgratz@fredhutch.org'
+    config_profile_url         = 'https://sciwiki.fredhutch.org/'
+    // Same as resourceLimits, but for older pipelines
+    max_memory = 700.GB
+    max_cpus = 36
+    max_time = 720.h
+    // Assumes user will provide a scratch dir at runtime
+    scratch = null
+}
+
+// If the user does not provide a scratch directory, use their home dir
+workDir = params.scratch ? "${params.scratch}" : "${System.getenv('HOME')}/nextflow"
```

Comment:
This is not exactly wrong but probably confusing for someone used to nextflow, is there a problem with the nextflow default behaviour?

Reply:
Obviously your choice, my hunch is that it's somewhat confusing compared to how most use (and e.g. what `nextflow clean` does and so on) but certainly workable.